salt/_modules/metalk8s_checks.py

```diff
@@ -434,6 +442,33 @@ def route_exists(destination, raises=True):
     return error or True
 
 
+def containerd_filesystem(folder=None, raises=True, **kwargs):
+    error = None
+    folder = folder or "/var/lib/containerd"
```

Comment:
TBH I don't think we really need this folder to be configurable, but as you wish